routes/voice-alexa.js

```diff
@@ -0,0 +1,498 @@
+'use strict';
+const debug = require('debug')('correlations-game:routes:voice');
+const express = require('express');
+const router = express.Router();
+const games = require('../bin/lib/game');
+const responses = require('../responses/content');
+const Alexa = require('alexa-sdk');
+
+const APP_ID = process.env.APP_ID;
+
+const spoor = require('../bin/lib/log-to-spoor');
```

Comment:
Ensure spoor tracking matches the latest that's on `master` for the Google route